packages/lb-annotation/src/core/toolOperation/basicToolOperation.ts

```diff
@@ -1021,6 +1023,10 @@ class BasicToolOperation extends EventListener {
     this.renderEnhance = renderEnhance;
   }
 
+  public setCustomRenderStyle(customRenderStyle: (rect: IRect) => IAnnotationStyle) {
```

Comment:
customRenderStyle: (data: IRect | IPolygon | IPoint | ILine | ITagResult | IBasicText) => IAnnotationStyle

Reply:
You can define a `basicType` like `type TAnnotationType = IRect | IPolygon | IPoint | ILine | ITagResult | IBasicText`.